kratos.gid/apps/Structural/xml/NodalConditions.xml

```diff
@@ -16,21 +16,21 @@
   </NodalConditionItem>
   
   <NodalConditionItem  n="PRESSURE" pn="Pressure" ProcessName="AssignScalarVariableProcess" VariableName="PRESSURE" ImplementedInApplication="StructuralMechanicsApplication"
-		       analysis_type="Static,Dynamic"  unit_magnitude="P"  units="Pa" v="Yes" App="Structural" state="CheckNodalConditionStateSolid" Interval="Total">
+		       analysis_type="Dynamic"  unit_magnitude="P"  units="Pa" v="Yes" App="Structural" state="CheckNodalConditionStateSolid" Interval="Total">
   </NodalConditionItem>
   <NodalConditionItem  n="CONTACT" pn="Contact" ProcessName="ALMContactProcess" VariableName="" ImplementedInApplication="ContactStructuralMechanicsApplication"
-		       analysis_type="Static,Dynamic"  App="Structural" state="" Interval="False"  ov="[CheckGeometry 2]"  ProductionReady="Developer">
+		       analysis_type="Static,Quasi-static,Dynamic"  App="Structural" state="" Interval="False"  ov="[CheckGeometry 2]"  ProductionReady="Developer">
   </NodalConditionItem>
   <NodalConditionItem n="ROTATION" pn="Rotation" ProcessName="AssignVectorVariableProcess" App="Structural" ImplementedInApplication="StructuralMechanicsApplication"
-					  analysis_type="Static,Dynamic" unit_magnitude="Angle" units="rad" VariableName="ROTATION" state="CheckNodalConditionStateSolid" Interval="Total">
+					  analysis_type="Static,Quasi-static,Dynamic" unit_magnitude="Angle" units="rad" VariableName="ROTATION" state="CheckNodalConditionStateSolid" Interval="Total">
     <outputs>
       <parameter n="TORQUE" pn="Torque" v="Yes"/>
     </outputs>
   </NodalConditionItem>
   <NodalConditionItem n="ANGULAR_VELOCITY" pn="Angular Velocity" ProcessName="AssignVectorVariableProcess" state="CheckNodalConditionStateSolid" Interval="Total"
-					  analysis_type="Dynamic" unit_magnitude="Angle/T" units="rad/s" VariableName="ANGULAR_VELOCITY" App="Structural" ImplementedInApplication="StructuralMechanicsApplication">
+					  analysis_type="Quasi-static,Dynamic" unit_magnitude="Angle/T" units="rad/s" VariableName="ANGULAR_VELOCITY" App="Structural" ImplementedInApplication="StructuralMechanicsApplication">
```

Comment:
Quasi-Static should not be added here, it is still a static analysis (with several time-steps)